scala-package/mxnet-demo/java-demo/pom.xml

```diff
@@ -21,7 +28,30 @@
                 <mxnet.version>[1.5.0-SNAPSHOT, )</mxnet.version>
             </properties>
         </profile>
-    </profiles>
+        <profile>
+          <id>osx-x86_64</id>
+          <activation>
+            <os>
+              <family>mac</family>
+            </os>
+          </activation>
+          <properties>
+            <mxnet.profile>osx-x86_64-cpu</mxnet.profile>
+          </properties>
+        </profile>
+        <profile>
+          <id>linux-x86_64</id>
+          <activation>
+          <os>
+            <family>unix</family>
+            <name>Linux</name>
+          </os>
+          </activation>
+          <properties>
+            <mxnet.profile>linux-x86_64-cpu</mxnet.profile>
```

Comment:
This will always pick the cpu artifact. The existing code allows one to pick up the gpu artifacts as well. 

https://github.com/apache/incubator-mxnet/blob/master/scala-package/mxnet-demo/scala-demo/Makefile#L35

Reply:
In order to simplify the process, I decided to get rid of the makefile entirely. To address your concern, I add a noteline in the README to inform then using GPU to run

Reply:
Yeah. removing makefile was a good riddance :) 

I'm fine with adding it to the README.